java/google/registry/rde/imports/RdeHostImportAction.java

```diff
@@ -117,7 +117,7 @@ private RdeImportUtils createRdeImportUtils() {
     public void map(JaxbFragment<XjcRdeHostElement> fragment) {
       final XjcRdeHost xjcHost = fragment.getInstance().getValue();
       try {
-        logger.infofmt("Converting xml for host %s", xjcHost.getName());
+        logger.infofmt("Starting transaction for host %s", xjcHost.getName());
```

Comment:
It seems to me like this logging statement is redundant with the "Saving host" one on line 123, as no actual work is being down in between.  Want to shave them down to a single logger call?